src/displayapp/screens/WatchFaceBinary.cpp

```diff
@@ -0,0 +1,693 @@
+#include "WatchFaceBinary.h"
+
+#include <date/date.h>
+#include <lvgl/lvgl.h>
+#include <cstdio>
+#include "BatteryIcon.h"
+#include "BleIcon.h"
+#include "NotificationIcon.h"
+#include "Symbols.h"
+#include "components/battery/BatteryController.h"
+#include "components/ble/BleController.h"
+#include "components/ble/NotificationManager.h"
+#include "components/heartrate/HeartRateController.h"
+#include "components/motion/MotionController.h"
+#include "components/settings/Settings.h"
+#include "../DisplayApp.h"
+
+using namespace Pinetime::Applications::Screens;
+
+WatchFaceBinary::WatchFaceBinary(DisplayApp* app,
+                                   Controllers::DateTime& dateTimeController,
+                                   Controllers::Battery& batteryController,
+                                   Controllers::Ble& bleController,
+                                   Controllers::NotificationManager& notificatioManager,
+                                   Controllers::Settings& settingsController,
+                                   Controllers::HeartRateController& heartRateController,
+                                   Controllers::MotionController& motionController)
+	: Screen(app),
+		currentDateTime {{}},
+		dateTimeController {dateTimeController},
+		batteryController {batteryController},
+		bleController {bleController},
+		notificatioManager {notificatioManager},
+		settingsController {settingsController},
+		heartRateController {heartRateController},
+		motionController {motionController}
+{
+	settingsController.SetClockFace(3);
+
+	displayedChar[0] = 0;
+	displayedChar[1] = 0;
+	displayedChar[2] = 0;
+	displayedChar[3] = 0;
+	displayedChar[4] = 0;
+	displayedChar[5] = 0;
+	displayedChar[6] = 0;
+
+
+	// !!! Achtung: alignments für alle folgenden Elemente ggfs im Hauptteil noch anpassen //
```

Comment:
Please translate your comments to english if they're necessary.

Reply:
i removed unneccesary comments and corrected the rest